CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### App Center
 
+* **[Feauture]** Add support for Apple Silicon (arm64 simulators are not supported via CocoaPods).
```

Comment:
The support is just for macOS modules right? We should specify this in each supported modules only then. Or mention  ''Add support....to existing macOS modules".

Reply:
Well to think of it, yeah, that makes sense. I'll update it